guides/v2.2/performance-best-practices/index.md

```diff
@@ -2,7 +2,9 @@
 group: performance-best-practices
 title: Performance Best Practices
 landing-page: Performance Best Practices
-redirect_from: /guides/v2.2/config-guide/prod/prod_perf-optimize.html
+redirect_from: 
```

Comment:
This is the file that should be renamed (moreso than the one you did). It's a top-level index file.

Also add the following line to the front matter:

permalink: performance-best-practices

Reply:
Don't add the permalink line yet. This has to wait until we complete the directory restructuring project

Reply:
Fixed, renamed the file, added redirect, updated symlink for 2.3